scripts/notify.lic

```diff
@@ -160,12 +440,12 @@ module Notify
   end
```

Comment:
`title = [message.from, Char.name].join(" => ")` is showing all notification as being "to" the character running the server thread. We probably need to save this onto `message` in `fmt_message` and update the title here to use `message.to`

Reply:
Good catch.  I pushed a commit that passes the real `Char.name` over tcp using `json` now